README.md

```diff
@@ -5,8 +5,10 @@ The ESLint generator for [Yeoman](https://yeoman.io/). This generator is intende
 ## Installation
 
 First and foremost, you must have [Node.js](https://nodejs.org/) and npm installed. If you don't have Node.js installed, please download and install the latest version.
-
-You must also install Yeoman, if you don't have it installed already. To install Yeoman, you can run this command:
+> **Requirements**
+>
+> - Node.js ^14.17.0 || ^16.0.0 || >= 18.0.0
+You must also install Yeoman, if you don't have it installed already.To install Yeoman, you can run this command:
```

Comment:
A blank line is needed above this sentence. Please take a look at the rendered version and you'll see it's showing up incorrectly.

Reply:
Done